src/themes/theme.js

```diff
@@ -0,0 +1,38 @@
+import tw from "twin.macro";
+
+const theme = {
+  button: {
+    default: {
+      base: tw`inline-flex items-center shadow-sm font-medium`,
+      color: tw`text-white-100 bg-indigo-600`,
+      hover: tw`hover:bg-indigo-700`,
+      focus: tw`focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500`,
+    },
+    color: {
+      primary: tw`bg-primary-500 hover:bg-primary-600 border-primary-500`,
+      success: tw`bg-success-500 hover:bg-success-600 border-success-500`,
+      danger: tw`bg-danger-600 hover:bg-danger-700 border-danger-600`,
+    },
+    border: {
+      gray: tw`border border-gray-300 text-gray-700 bg-white-100 hover:bg-gray-50`,
+      grey: tw`border border-gray-300 text-gray-700 bg-white-100 hover:bg-gray-50`,
```

Comment:
Is there a way that we can make this DRYer?  There are two ways to write the word "grey" (with an "a" or with an "e").  I want to cover both possibilities.  For now, I just duplicated the code twice -- is there a better way to do this?

Reply:
@MaksymTaratuta55, only add it as constant:
```
const someConstant = tw`border border-gray-300 text-gray-700 bg-white-100 hover:bg-gray-50`;

{
...,
gray: someConstant,
grey: someConstant
}
```

Reply:
fixed